tests/fixtures/saas/hubspot_fixtures.py

```diff
@@ -85,3 +95,45 @@ def dataset_config_hubspot(
     )
     yield dataset
     dataset.delete(db=db)
+
+
+@pytest.fixture(scope="function")
+def setup_teardown_erasure_hubspot_data(connection_config_hubspot, hubspot_erasure_identity_email) -> Generator:
```

Comment:
Nit: All fixtures setup and teardown, this might be better named `hubspot_erasure_data`